.gitignore

```diff
@@ -5,3 +5,5 @@
 /local.properties
 *.iml
 .DS_Store
+
+!gradle-wrapper.jar
```

Comment:
Do not actually remember how it got there. I guess it may be generated automatically. Maybe I've added it for some purpose.